Criollo/Source/FCGI/CRFCGIResponse.m

```diff
@@ -82,7 +82,7 @@ - (void)buildHeaders {
 - (void)writeData:(NSData *)data finish:(BOOL)flag {
 
     if ( self.finished ) {
-        @throw [NSException exceptionWithName:NSInternalInconsistencyException reason:@"Response is already finished" userInfo:nil];
```

Comment:
This is on purpose. On should never get into a state where data is being sent after the response has finished.  It's mean to point out application logic errors.

We should leave it as it.

Reply:
That's exactly the point. This should be handled in the framework, i.e. it should not get itself into that state. I'll also look into it, but i'm thinking the best place to look at it is somewhere in the static file serving logic.

The question is: what should we do with files that are constantly changing? Lock them while we're reading? Gracefully stop sending if an EOF has been encountered prior to finishing? I'd try to find the root cause of the issue first and get a better understanding of the problem, then decide on the best solution.

I'm curious what apache/nginx et all do?